slw_agent/aea-config.yaml

```diff
@@ -0,0 +1 @@
+/home/solarw/MyData/work/hyperion/autonolas/slw_service/packages/slw/agents/slw_agent/aea-config.yaml
```

Comment:
I would suggest publishing this agent and it's components to the local packages directory, committing the whole agent will introduce a lot of code duplication.

Reply:
@angrybayblade was done for dev purposes.

as I got some commands can be executed only within agent scope.
this is why agent dir present. 

now it excluded from the git index